### DISCUSSION TITLE
请问是否能提供以前的$5以及更小面值的充值入口？

### DISCUSSION THREAD

**Discussion by NickCN47** [BurnHair/docs#176] [分享讨论]
我看现在只有$10和$15，我记得以前有$1,$3,$5的，请问还会补充库存吗？

**Comment by BennyThink** [comment]
都点开刷完，会自动增加3和5

**Reply by axdlee** [reply]
赶紧花完，马上要清空付费额度了

**Reply by BennyThink** [reply]
<img width="437" alt="image" src="https://github.com/user-attachments/assets/75a4105a-b0d8-4907-b931-02aabc7a93ec">

😆

**Reply by NickCN47** [reply]
请问我今天不科学打不开nextchat，科学之后能打开但是还是使用不了，回复报的error格式如下：
{
  "error": {
    "message": " (request id: *************************",
    "type": "upstream_error",
    "param": "500",
    "code": "bad_response_status_code"
  }
}
请问如何解决？换了很多个节点也新建了很多个对话了，仍然如此....

**Reply by BennyThink** [reply]
你用的模型是哪个？用的是burn.hair 还是cn-test.burn.hair ?

**Reply by NickCN47** [reply]
burn.hair，直接用的主页跳转的nextchat，模型用的4o

**Reply by BennyThink** [reply]
<img width="676" alt="image" src="https://github.com/user-attachments/assets/45e65f1f-cee1-4c8a-92a3-5acba2995be4">

没问题啊🤨 你试试 cn-test?

**Reply by axdlee** [reply]
垃圾
